src/admin/dto/create-admin.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { IsEmail, IsNotEmpty, IsString } from "class-validator";
+
+export class CreateAdminDto {
+  @IsNotEmpty()
+  @IsEmail()
+  readonly email: string;
+
+  @IsNotEmpty()
+  @IsString()
+  readonly password: string;
+
+  readonly hashedRefreshToken: string;
+
+  readonly createdAt: Date;
+
+  readonly updatedAt: Date;
+
+  @IsNotEmpty()
+  readonly permission: string;
+
+  readonly isDeleted: boolean;
```

Comment:
I believe you can simply delete `readonly isDeleted: boolean;` . When you create a new admin account, it must be isDeleted: false.

Reply:
deleted